echo-pubsub-core/echo-pubsub-core.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
   implementation "com.squareup.retrofit:retrofit"
   implementation "com.squareup.retrofit:converter-jackson"
   implementation "com.jakewharton.retrofit:retrofit1-okhttp3-client:1.1.0"
+  implementation files('cdevents-sdk-java-0.0.1.jar')
```

Comment:
This local file can be removed and used from maven repo once this PR is fixed, https://github.com/cdevents/sdk-java/pull/9